### DISCUSSION TITLE
Insert generated response Not Working

### DISCUSSION THREAD

**Discussion by AlanTsai0329** [GorvGoyl/ChatGPT-Writer#117] [Report Bugs and Issues]
Insert generated response not working after I generated email.

**Comment by GorvGoyl** [comment]
Fixed in today's release.

**Reply by AlanTsai0329** [reply]
It's worked today, thanks.

**Comment by reemshareef** [comment]
its showing to me like this Please go to a regular website like Gmail and then launch this extension.

**Comment by GorvGoyl** [comment]
Please refresh the gmail tab and it should work.

**Comment by reemshareef** [comment]
I refresh many time but still same issue

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by GorvGoyl** [comment]
Could you please share a screenshot so that i can look into it.

**Comment by reemshareef** [comment]
see when i put the content there is no response generated

________________________________

**Comment by GorvGoyl** [comment]
Hello, we've pinpointed the problem and are actively working on a solution.
We'll have it resolved as quickly as we can.

**Comment by reemshareef** [comment]
Iam waiting your updates

________________________________

**Comment by reemshareef** [comment]
no dear still not working i just now try but still not working
________________________________

**Comment by GorvGoyl** [comment]
Hi, we've pinpointed the problem and are actively working on a solution.
We'll have it resolved as quickly as we can.
